src/profile/d3Elevation.js

```diff
@@ -58,7 +59,7 @@ goog.require('ol.obj');
  * @param {ngeox.profile.ProfileOptions} options Profile options.
  * @export
  */
-ngeo.profile = function(options) {
+ngeo.profile.d3Elevation = function(options) {
```

Comment:
It is a constructor so it should start uppercase.

Reply:
It is ? or it is a big function, but not a constructor ?

Reply:
There is the annotation in the function comment.

Reply:
I think the constructor (never used with new) and struct (doesn't have any properties on itself) annotations are incorrect here.

Reply:
I've removed the - * (at)constructor  - * (at)struct tags